### PR TITLE
Upgraded all the packages to latest version.

### PR DESCRIPTION
**Pull request by radar101**
Upgrade all packages to the last version [#665](https://github.com/CyBear-Jinni/cbj_app/issues/665)

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -51,9 +51,6 @@
                  screen fades out. A splash screen is useful to avoid any visual
                  gap between the end of Android's launch screen and the painting of
                  Flutter's first frame. -->
```
Comment (guyluz11) [inline android/app/src/main/AndroidManifest.xml:53]: Why have you deleted this file?.
Also this comment it part of the line that you deleted, it should get removed with it

File: lib/injection.dart [hunk 2]
```diff
@@ -3,7 +3,7 @@ import 'package:cybear_jinni/utils.dart';
 import 'package:get_it/get_it.dart';
 import 'package:injectable/injectable.dart';
 
-final getIt = GetIt.instance;
+GetIt getIt = GetIt.instance;
```
Comment (guyluz11) [inline lib/injection.dart:6]: Can it be `final GetIt`?

File: pubspec.yaml [hunk 3]
```diff
@@ -45,6 +45,8 @@ dependencies:
   auto_size_text: ^3.0.0
   # Helps implement the BLoC pattern.
   bloc: ^8.1.2
+
```
Comment (guyluz11) [inline pubspec.yaml:48]: Remove extra space

File: pubspec.yaml [hunk 4]
```diff
@@ -98,6 +100,8 @@ dependencies:
   iconify_flutter: ^0.0.5
   # Convenient code generator for get_it
   injectable: ^2.3.0
+  injectable_generator: ^2.1.3 # Version 2
+
```
Comment (guyluz11) [inline pubspec.yaml:104]: Remove extra space

File: pubspec.yaml [hunk 5]
```diff
@@ -98,6 +100,8 @@ dependencies:
   iconify_flutter: ^0.0.5
   # Convenient code generator for get_it
   injectable: ^2.3.0
+  injectable_generator: ^2.1.3 # Version 2
```
Comment (guyluz11) [inline pubspec.yaml:103]: `injectable_generator` already exists in `dev_dependencies`.
Please update the version there

**Review by guyluz11 — CHANGES_REQUESTED** [review]

**Review by radar101 — COMMENTED** [review]
Made the required changes. Please review them.

File: pubspec.yaml (hunk 5 above)
Comment (guyluz11) [reply pubspec.yaml]: You removed it entirely instead of updating it.
Please update the existing one

**Comment by radar101** [comment]
I've deleted it because the updated package was mentioned in dev_dependendies as
injectable_generator: ^2.1.5

**Comment by guyluz11** [comment]
Thanks for creating the pr

I have decided to take this pr https://github.com/CyBear-Jinni/cbj_app/pull/679 because it included update for all of the packages.

**Pull request closed without merging**